crates/lib3h/src/engine/real_engine.rs

```diff
@@ -465,10 +513,25 @@ impl<T: Transport, D: Dht, SecBuf: Buffer, Crypto: CryptoSystem> RealEngine<T, D
                 timestamp: 42, // FIXME
             }),
         )?;
+        // Send Get*Lists requests
+        let mut output = Vec::new();
+        let mut list_data = GetListData {
+            space_address: join_msg.space_address.clone(),
+            provider_agent_id: join_msg.agent_id.clone(),
+            request_id: "gossiping".to_owned(),
+        };
+        output.push(Lib3hServerProtocol::HandleGetGossipingEntryList(
+            list_data.clone(),
+        ));
+        list_data.request_id = "authoring".to_owned();
+        output.push(Lib3hServerProtocol::HandleGetAuthoringEntryList(list_data));
         // Done
-        Ok(Lib3hServerProtocol::SuccessResult(res))
+        output.push(Lib3hServerProtocol::SuccessResult(res));
```

Comment:
I realize it shouldn't actually matter, but do you think we should respond with the success to the JoinSpace request first, before making the list requests?

Reply:
Good point